.github/workflows/pull_requests.yml

```diff
@@ -179,9 +195,19 @@ jobs:
         with:
           go-version: "1.20"
           cache-dependency-path: "**/go.sum"
-      - name: test
+      - name: Setup python
+        if: needs.scripts-changed.outputs.any_changed == 'true'
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: test templates
         if: needs.chart-changed.outputs.any_changed == 'true'
         run: make test-templates
+      - name: test python scripts
+        if: needs.scripts-changed.outputs.any_changed == 'true'
+        run: |
+          pip install pyyaml
+          make test-python-scripts
```

Comment:
Can we make a separate job for this? I don't want to get test failures because there's something wrong with a helper script we only use for troubleshooting.